src/main/java/sage/Main.java

```diff
@@ -0,0 +1,33 @@
+package sage;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
```

Comment:
I like how the imports are mostly very specific and follow an alphabetical order, but perhaps a few of these slight switches can be made to ensure the whole repo follows the same standard? (Just a nit!)